README.md

```diff
@@ -67,7 +67,7 @@ After writing this method, our new result should be:
 ...
 ```
 
-**think**: Where should we call `strip_punctuation`? Does it matter? Why?
+**think**: Where should we call `strip_punctuation`? Does it matter? Why? We need to call strip_punctuation before we call analyze_mood, because in order for analyze_mood to populate correct results the strings must be stripped of their text.
```

Comment:
This is true. What I should have asked is "Should you call strip_punctuation inside or outside the analyze_mood method?"